support-frontend/assets/helpers/redux/subscriptionsStore.ts

```diff
@@ -39,42 +43,41 @@ export const startSubscriptionsListening =
 
 const subscriptionsStore = configureStore({
 	reducer: baseReducer,
-	// Makes devtools work correctly with the re-created store
-	devTools: false,
 	middleware: (getDefaultMiddleware) =>
 		getDefaultMiddleware().prepend(listenerMiddleware.middleware),
 });
 
 export type SubscriptionsStore = typeof subscriptionsStore;
 
-export function addPageReducer(
```

Comment:
Because all of the information we used to pass into `createReducer` is now handled by the product and address slices, and the base subs checkout state is the same for all products, we can get rid of this horrible fudge and simplify the store setup.